test/language/expressions/arrow-function/arrow/arrow-body-in.js

```diff
@@ -0,0 +1,10 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: Arrow body is in non strict mode
+flags: [noStrict]
+es6id: 14.2
+---*/
+
+(() => eval++)();

```

Comment:
This is a duplicate of https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/syntax/arrowparameters-bindingidentifier-eval.js